int16[] data
int16 doa
float64 start_time
float64 end_time